packages/vite/src/node/server/index.ts

```diff
@@ -152,7 +153,7 @@ export interface FileSystemServeOptions {
    * This will have higher priority than `allow`.
    * picomatch patterns are supported.
    *
-   * @default ['.env', '.env.*', '*.crt', '*.pem']
+   * @default ['.env', '.env.*', '*.\{crt,pem\}']
```

Comment:
I might be wrong, but I think this was done because `\` get rendered in IDE tooltips directly instead of working as escaping char, so pls revert this